### PR TITLE
Added ability to get geo codes from place id or address

### PR DESCRIPTION
**Pull request by imbudhiraja**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -251,3 +251,97 @@ Example:
...
+ */
+getLatLng(result);
+```
+
+#### result
+
+Type: `Object`
+Required: `true`
+
+One of the element from `results` (returned from Google Maps Geocoder)
```
Comment (Tintef) [inline README.md]: Typo: `One of the elements from...`

File: src/utils/google-geocodes-helper [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    });
+  });
+};
+
+export const getLatLng = result => {
+  return new Promise((resolve, reject) => {
+    try {
+      const latLng = {
+        lat: result.geometry.location.lat(),
+        lng: result.geometry.location.lng()
```
Comment (Tintef) [inline src/utils/google-geocodes-helper]: Add a trailing comma here 😃

**Comment by imbudhiraja** [comment]
Hey @Tintef , 
Thanks for sharing feedback. I have fixed issues pointed out by you in PR.

**Comment by Tintef** [comment]
@imbudhiraja Awesome, thanks for fixing that stuff!

I'll give a try on the weekend and merge it afterwards!

File: README.md [hunk 3]
```diff
@@ -251,3 +251,144 @@ Example:
...
+
+#### address
+
+Type: `String`,
+Required: `true`
+
+String that gets passed to Google Maps [Geocoder](https://developers.google.com/maps/documentation/javascript/geocoding)
+
+```js
+import { geocodeByAddress } from 'react-places-autocomplete';
```
Comment (Tintef) [inline README.md]: The name of the package is `react-google-places-autocomplete`, please change it

File: README.md [hunk 4]
```diff
@@ -251,3 +251,144 @@ Example:
...
+
+#### placeId
+
+Type: `String`,
+Required: `true`
+
+String that gets passed to Google Maps [Geocoder](https://developers.google.com/maps/documentation/javascript/geocoding)
+
+```js
+import { geocodeByPlaceId } from 'react-places-autocomplete';
```
Comment (Tintef) [inline README.md]: Same here

File: README.md [hunk 5]
```diff
@@ -251,3 +251,144 @@ Example:
...
+
+#### result
+
+Type: `Object`
+Required: `true`
+
+One of the elements from `results` (returned from Google Maps Geocoder)
+
+```js
+import { geocodeByAddress, getLatLng } from 'react-places-autocomplete';
```
Comment (Tintef) [inline README.md]: Same here

**Comment by imbudhiraja** [comment]
@Tintef updated

**Comment by Tintef** [comment]
@imbudhiraja I changed to base branch to `develop` as I won't upload this to npm registry until next week

**Pull request merged**